.env.example

```
@@ -46,12 +50,24 @@ PLAYWRIGHT_MICROSERVICE_URL="http://taboot-playwright:3000/scrape"
 REDIS_RATE_LIMIT_URL="redis://taboot-cache:6379"
 # Redis Connection Pooling
 REDIS_MAX_CONNECTIONS="100"
 REDIS_SOCKET_TIMEOUT="10"
 
+# TRUST_PROXY: Set to 'true' ONLY if behind trusted reverse proxy
+# Examples: Cloudflare, nginx, AWS ALB, Vercel
+# SECURITY: Never set to 'true' if exposed directly to internet
+# When true: Trusts X-Forwarded-For header for client IP (validates format)
+# When false (default): Ignores X-Forwarded-For, uses connection IP
+# Default: 'false' (doesn't trust X-Forwarded-For)
+TRUST_PROXY="false"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)